pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -241,17 +241,17 @@ func (c *Controller) syncOwnedProvisionRequest(
 			continue
 		}
 
-		oldPr, exists := activeOrLastPRForChecks[checkName]
+		req, exists := activeOrLastPRForChecks[checkName]
```

Comment:
why is this renamed? I think the previous name makes sense as this is a pre-existing provreq. Also, it feels like extending the diff unnecessarily for a bugfix PR which is going to be cherry-picked.

Reply:
Ah, I see you use it later for `syncProvisionRequestsPodTemplates`, this makes sense to reuse the same name for the new and old provReq. Feel free to keep as is then.